igvc_navigation/src/mapper/wrapper_layer.cpp

```diff
@@ -32,6 +32,7 @@ void WrapperLayer::updateCosts(costmap_2d::Costmap2D& master_grid, int min_i, in
   std::optional<cv::Mat> map = mapper_.mapper_->getMap();
   if (!map)
   {
+    current_ = false;
```

Comment:
Leftover from debugging?

Reply:
This was intentional, because MBF (or the planners, I don't remember exactly) decides if it can or can't use a map based on whether it's current.